application/pom.xml

```diff
@@ -261,6 +266,22 @@
             <groupId>org.hsqldb</groupId>
             <artifactId>hsqldb</artifactId>
         </dependency>
+        <dependency>
+            <groupId>eu.infomas</groupId>
+            <artifactId>annotation-detector</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>com.hashmap.tempus</groupId>
+            <artifactId>annotations</artifactId>
+            <version>1.0-SNAPSHOT</version>
+        </dependency>
+        <dependency>
+            <groupId>com.sun</groupId>
+            <artifactId>tools</artifactId>
+            <version>1.8</version>
+            <scope>system</scope>
+            <systemPath>${java.home}/../lib/tools.jar</systemPath>
+        </dependency>
```

Comment:
This dependency is required if we need to Compile Java classes at runtime. I need to still find an alternative as tools.jar may not be available from Java 9